projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/task_runner.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import threading
+
+
+class TaskRunner:
+    def __init__(self):
```

Comment:
A doc string maybe?

Reply:
Here we definitely need a good docstring

Reply:
As far as I see this class is responsible only for running a single task right? 
Do we want it to handle multiple tasks? Maybe we need to talk it over again

Reply:
Is there a use case for running multiple tasks from a single notebook? If yes, then this should definitely be updated to poll multiple tasks.

Reply:
I don't think I got this out of the discussion prior to the implementation. I thought we want to support single operation at a time. I guess it's a matter of when do we want to start handling multiple tasks but I think for now handling one should work just well and then update it when we want to handle multiple tasks.
@antoniivanov what is your view on this?

Reply:
I'd suggest we go with a first iteration of supporting 1 task at a time and improve it as a next step.